backend/frontend/src/App.js

```diff
@@ -13,11 +13,33 @@ const theme = createTheme({
   },
 });
 
+const customTheme = createTheme({
+  palette: {
+    primary: {
```

Comment:
Just asking
We have `default` and `custom` themes in the code.
Do we use these `primary` and `secondary` definitions later?

Reply:
I thought we did, but turns out we don't. Removing